Release-Notes/Release3.md

```diff
@@ -72,25 +75,38 @@ Per cluster application credentials.
 
 ### OSISM
 
-## Removals
+* In ``environments/kolla/secrets.yml`` the parameter ``neutron_ssh_key`` must be
+  added.
+
+  ```
+  neutron_ssh_key:
+    private_key:
+    public_key:
+  ```
 
-* OpenStack Wallaby (?) images are no longer built and thus no longer kept updated
+  The ssh key can be generated as follows: ``ssh-keygen -t rsa -b 4096 -N "" -f id_rsa.neutron -C "" -m PEM``
 
-CHECK WHICH OLD deprecations have resulted in removals ...
+## Removals
 
-* Cockpit is deprecated in favor of Boundary by HashiCorp or Teleport
-* ceph-ansible is deprecated in preparation for cephadm
+* The Cockpit service has been removed.
 * All osism- scripts on the manager are deprecated and will be replaced by the new OSISM CLI. The scripts will be removed in the next release
-* The following services are currently not used and are deprecated and scheduled for removal as of now: Falco, Jenkins, Rundeck, Lynis, Trivy
-* Heat will no longer be offered by default in the testbed in the future
-* The docker-compose CLI will be removed and replaced by the new compose plugin for Docker.
-docker-compose is then no longer available and docker compose must be used instead
 
 ## Deprecations
 
 Deprecations happen according to our [deprecation policy](https://github.com/SovereignCloudStack/Docs/blob/main/Design-Docs/Release-Policies.md#deprecation).
 
-What ... ?
+* Linux bridge support has been deprecated by the Neutron team and marked as experimental.
+  If you are using it in your deployment, we suggest to migrate to OpenVSwitch instead.
+* Debian dropped hddtemp (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1002484),
+  therefore we will remove the ``hddtemp`` service in the next release, as there is
```

Comment:
I'd suggest a different wording:

"If Linux bridge is used in deployments, migrating to OpenVSwitch is recommended".

For the hddtemp comment:

"therefore ``hddtemp`` will be removed from the next OSISM release"